### ISSUE TITLE
No module named google.protobuf.internal

### ISSUE THREAD

**Issue by palmerj** [geometalab/Vector-Tiles-Reader-QGIS-Plugin_2016#35]
On MacOSX when using 2.14.2 I can't get the plugin to load. I downloaded the Vector-Tiles-Reader-QGIS-Plugin zip from github and installed into ~/.qgis2/python/plugins, but when I try to load the plugin I get the following error:

```
This plugin is broken
No module named google.protobuf.internal
```

**Comment by sfkeller** [comment]
Thanks for the report. This bug under MacOSX is probably same as #18 for Windows. Note that this is work in (slow) progress! Expected release as experimental code is around June 2017.

**Comment by palmerj** [comment]
Thanks. I just tried on Ubuntu and got the same message. Are there any python libraries I need installed first?

**Comment by sfkeller** [comment]
Yes; those Protobuf. But currently don't have time to make a fix - that's why release is scheduled for next year. 
Just wondering: What's the case for which you'd like to use this plugin? If it's just as base map then WMTS (e.g. QuickmapServices) is probably better suited, since this plugin delivers vector tiles as geometries which need to be symbolized separately.

**Comment by palmerj** [comment]
I will like to do some visualisation of mbtiles files generated vtiles from tippecanoe. So it's not possible to do wmts unless I use something like tileserver-gl.
